solutions/remotemonitoring/armtemplates/basic-parameters.json

```diff
@@ -44,9 +44,6 @@
     "deploymentId": {
       "value": "{Added through command line interface}"
     },
-    "diagnosticsEndpointUrl": {
-      "value": "{Added through command line interface}"
-    },
     "userPrincipalObjectId": {
```

Comment:
Add the applicationInsightsInstrumentationKey parameter here to match other params in this file

Reply:
Added.